portality/bll/services/concurrency_prevention.py

```diff
@@ -0,0 +1,14 @@
+from portality.core import app
+from portality.bll.exceptions import ConcurrentUpdateRequestException
+from portality.ui.messages import Messages
+import redis
+
+class ConcurrencyPreventionService:
+    def __init__(self):
+        self.rc = redis.Redis(host=app.config.get("HUEY_REDIS_HOST"), port=app.config.get("HUEY_REDIS_PORT"))
+
+    def prevent_concurrency(self, journal, id):
```

Comment:
Since we're going to do this for articles, I'd make this method name specific to what kind of concurrency

Reply:
I was thinking about it, but wouldn't that be possible (better) to make this service very general and then specify details on another level?

Reply:
If that's the case you need to think a bit more about the generality of this function.  It takes a `journal` and an `id` but that won't be true of other concurrency requests.  Similarly they won't all raise `ConcurrentUpdateRequestException`s.  So either the function needs to be more general, or you need custom functions for other use cases.

Because we haven't explored the other use cases yet, I'd fall back to having a more specific function for this particular use case, and then we may be able to refactor later when we know about articles in more detail.